packages/thirdweb/src/react/web/ui/ConnectWallet/screens/SendFunds.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { describe, expect, it, vi } from "vitest";
+import { render } from "~test/react-render.js";
+import { TEST_CLIENT } from "~test/test-clients.js";
+import { TEST_ACCOUNT_A } from "~test/test-wallets.js";
+import { base } from "../../../../../chains/chain-definitions/base.js";
+import { useActiveWalletChain } from "../../../../../react/core/hooks/wallets/useActiveWalletChain.js";
+import en from "../locale/en.js";
+import { SendFundsForm } from "./SendFunds.js";
+
+const client = TEST_CLIENT;
+
+describe("SendFunds screen", () => {
```

Comment:
thank you for the tests! 76% coverage really nice 👌

Reply:
yesser, just trying to push to 80% so we can get this PR merged!